src/store/actions.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  COMMIT_TO_LIST ({ commit }, payload) {
+    // Fake asynchronous call to server
+    return setTimeout(() => {
```

Comment:
this is fine for now - we should replace with call to local server later

Reply:
OK